# Generated by Django 3.1.2 on 2020-11-04 14:52

from django.conf import settings
import django.core.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0017_auto_20201104_1515'),
    ]

    operations = [
        migrations.AddField(
            model_name='takeaway',
            name='user',
            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
        ),
        migrations.AlterField(
            model_name='user',
            name='tel',
            field=models.CharField(max_length=10, validators=[django.core.validators.RegexValidator(message='Numero di telefono errato!', regex='^\\+?1?\\d{10,10}$')], verbose_name='Telefono:'),
        ),
    ]
